### ISSUE TITLE
Menu layout broken, missing options in Firefox 51

### ISSUE THREAD

**Issue by rombert** [gnome-integration-team/firefox-gnome#425]
The menu is completely unstyled, missing icons. For instance, the _Help_ section is unreachable.

![firefox-gnome-unstyled menu](https://cloud.githubusercontent.com/assets/21237/23302391/34d3ab68-fa97-11e6-91b3-8454c43e07c5.png)

**Comment by d9h02f** [comment]
You can use this userstyle for that: https://userstyles.org/styles/115022/gnome-styled-menu-list-view

**Comment by rombert** [comment]
Right, but I see this as a bug of the extension :-) and would rather not install user styles.

**Comment by NGRhodes** [comment]
I am currently using https://addons.mozilla.org/en-GB/firefox/addon/gnome-3-updated/ in FF52.
